assets/js/params/Custom/Code/AceEditor.js

```diff
@@ -17,7 +18,7 @@ export default class WorkbenchAceEditor extends React.PureComponent {
   static propTypes = {
     // When isZenMode changes, we'll call componentDidUpdate()
     isZenMode: PropTypes.bool.isRequired,
-    wfModuleOutputError: PropTypes.string, // hopefully null or empty
+    wfModuleOutputErrors: PropTypes.arrayOf(PropTypes.shape({ message: PropTypes.string.isRequired, quickFixes: PropTypes.arrayOf(PropTypes.shape(QuickFixPropTypes)).isRequired }).isRequired).isRequired, // may (hopefully) be empty
```

Comment:
Gah, now we're reusing this big block that has an error :).

I'll merge this and then refactor the prop types. (QuickFixPropTypes follows a bad convention, for a bad reason.) I should have done this ages ago.